### ISSUE TITLE
PowerShell Durable Templates still say preview

### ISSUE THREAD

**Issue by JustinGrote** [microsoft/vscode-azurefunctions#3010]
Issue Type: <b>Bug</b>

The Powershell Durable Tasks are GA but they still say preview in the Create Function Dialog

Extension version: 1.6.0
VS Code version: Code - Insiders 1.63.0-insider (d18d093403b12a65350c58a7b0d5771cc1f42aba, 2021-11-26T08:21:30.810Z)
OS version: Windows_NT x64 10.0.22000
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 4700U with Radeon Graphics          (8 x 1996)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.36GB (2.50GB free)|
|Process Argv|--crash-reporter-id 4453efaa-0821-4555-88e2-3de8bad9109e|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30258533
pythonptprofiler:30281269
vshan820:30294714
pythondataviewer:30285072
vscod805:30301674
pythonvspyt200:30323110
bridge0708:30335490
bridge0723:30353136
pythonrunftest32:30365365
pythonf5test824:30361779
javagetstartedt:30350119
pythonvspyt187:30365360
vsaa593:30376534
vsc1dst:30396469
pythonvs932:30404738
vscexrecpromptt3:30397560
vscop804cf:30404767
vs360:30404995

```

</details>

**Comment by nturinski** [comment]
@anthonychu I think that this is coming from the template feed.  Do you know where we would file an issue about this?

**Comment by anthonychu** [comment]
Yeah it’s here: https://github.com/Azure/azure-functions-templates/blob/4352bb2f5c7489e2a8111249bb3ac0c4ac7398c0/Functions.Templates/Templates/DurableFunctionsActivity-PowerShell-2.x/metadata.json#L4

I don’t have the powers to transfer this issue there. Please create a new one or we can use this one to track for now.
